.github/workflows/mirror.yml

```diff
@@ -0,0 +1,34 @@
+name: mirror
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - src/lib.rs
+
+permissions:
+  contents: write
+
+jobs:
+  readme:
+    name: readme
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3.4.0
```

Comment:
I know, I have some repositories on this account with pull requests from Dependabot. But they are code to me, that I do not maintain anymore, so I just ignore it. The same is true for this repository, I think. Right now I have no use case for this tool and I do not plan to maintain it.

Reply:
I also have some coding utilities which are currently unmaintained.  At the moment, I try to move them successively into aeruginous because these tools are mostly too simple to reason the maintenance overhead for their projects.  In aeruginous, they receive an own calling mode with easy to instantiate IO operations and obviously correct closures as function bodies.  I know that this violates against the UNIX design principle "Small is Beautiful" but in case I need these tools, I can rely on them being maintained as aeruginous is a larger project and state-of-the-art maintenance automations.  Furthermore, I just need to install **one** application without strange software requirements (e.g. GNU Octave for the build routine).

I can also prepare everything for moving commit-analyzer into aeruginous, if you wish.  This would prevent this project from dying due to being unmaintained.